worlds/civ_6/__init__.py

```diff
@@ -0,0 +1,184 @@
+    launch_subprocess(main, name="Civ6Client")
+
+
+components.append(
+    Component("Civ6 Client", func=run_client, component_type=Type.CLIENT,
+              file_identifier=SuffixIdentifier(".apcivvi"))
+)
+
+
+class CivVIWeb(WebWorld):
+    tutorials = [Tutorial(
+        "Multiworld Setup Guide",
+        "A guide to setting up Civlization VI for MultiWorld.",
+        "English",
+        "setup_en.md",
+        "setup/en",
+        ["hesto2"]
+    )]
+    theme = "ocean"
+
+
+class CivVIWorld(World):
+    """
+    Civilization VI is a turn-based strategy video game in which one or more players compete alongside computer-controlled opponents to grow their individual civilization from a small tribe to control the entire planet across several periods of development.
+    """
+
+    game = "Civilization VI"
+    topology_present = False
+    options_dataclass = CivVIOptions
+    options: CivVIOptions
+
+    web = CivVIWeb()
+
+    item_name_to_id = {
+        item.name: item.code for item in generate_item_table().values()}
+    location_name_to_id = {
+        location.name: location.code for location in generate_flat_location_table().values()}
+
+    item_table: Dict[str, CivVIItemData] = {}
+    location_by_era: Dict[EraType, Dict[str, CivVILocationData]]
+
+    required_client_version = (0, 4, 5)
+
+    def __init__(self, multiworld: "MultiWorld", player: int):
+        super().__init__(multiworld, player)
+        self.location_by_era = generate_era_location_table()
+
+        self.location_table: Dict[str, CivVILocationData] = {}
+        self.item_table = generate_item_table()
+
+        for _era, locations in self.location_by_era.items():
+            for _item_name, location in locations.items():
+                self.location_table[location.name] = location
+
+    def get_filler_item_name(self) -> str:
+        return get_random_filler_by_rarity(self, FillerItemRarity.COMMON).name
+
+    def create_regions(self) -> None:
+        create_regions(self, self.options, self.player)
+
+    def set_rules(self) -> None:
+        if self.options.boostsanity:
+            create_boost_rules(self)
+
+    def create_item(self, name: str) -> Item:
+        item: CivVIItemData = self.item_table[name]
+        classification = item.classification
+        if self.options.boostsanity:
+            if item.civ_name in BOOSTSANITY_PROGRESSION_ITEMS:
+                classification = ItemClassification.progression
+
+        return CivVIItem(item, self.player, classification)
+
+    def create_items(self) -> None:
+        progressive_era_item = None
+        for item_name, data in self.item_table.items():
+            # These item types are handled individually
+            if data.item_type in [CivVICheckType.PROGRESSIVE_DISTRICT, CivVICheckType.ERA, CivVICheckType.GOODY]:
+                continue
+
+            # If we're using progressive districts, we need to check if we need to create a different item instead

```

Comment:
```suggestion
        for item_name, data in self.item_table.items():
            # These item types are handled individually
            if data.item_type in [CivVICheckType.PROGRESSIVE_DISTRICT, CivVICheckType.ERA, CivVICheckType.GOODY]:
                continue
```
Looks like these two are leftover?

Reply:
Ah yep